README.md

```diff
@@ -25,13 +25,18 @@ Here are some examples of how you can use the PV211 Utils library:
   [![Open in Colab][colab-badge]][cranfield]
   [![Open in Jupyter Hub][jupyter-badge]][jupyter]
 
-- Second Term Project: TREC Collection (10.37% MAP score)
-  [![Open in Colab][colab-badge]][trec]
+- Second Term Project: Beir CQADupStack Collection (17.60% MAP score)
+  [![Open in Colab][colab-badge]][beir]
   [![Open in Jupyter Hub][jupyter-badge]][jupyter]
 
 - Alternative Second Term Project: ARQMath Collection (0.71% MAP score)
   [![Open in Colab][colab-badge]][arqmath]
   [![Open in Jupyter Hub][jupyter-badge]][jupyter]
+ 
+- Old Second Term Project: TREC Collection (10.37% MAP score)
```

Comment:
```suggestion
- Pre-2023 Second Term Project: TREC Collection (10.37% MAP score)
```

Reply:
The MAP score is also suspect and should be measured again after all the planned changes (using `TfIdfSystem`, change of MAP to MAP@k, etc.).